package/MDAnalysis/analysis/base.py

```diff
@@ -390,7 +390,7 @@ def _conclude(self):
         pass  # pylint: disable=unnecessary-pass
 
     def run(self, start=None, stop=None, step=None, frames=None,
-            verbose=None):
+            verbose=None, *, pbar_kwargs={}):
```

Comment:
I would like the name to be `progressbar_kwargs` to be slightly clearer.

Reply:
Fixed!